src/vl-radio.js

```diff
@@ -36,9 +36,11 @@ export class VlRadio extends VlElement(HTMLElement) {
       </style>
       
       <label class="vl-radio" for="radio">
-        <input class="vl-radio__toggle" type="radio" id="radio" name="radio"/>
+        <input class="vl-radio__toggle" type="radio" id="radio" data-vl-name="radio"/>
         <div class="vl-radio__label">
-          <span id="label-text"></span>
+          <span id="label-text">
+            <slot></slot>
```

Comment:
Misschien zit het niet tussen de use cases waarmee we rekening houden, maar wat als we eerst een text zetten via het attribuut en nadien via een slot iets mee willen geven. Doet die textContent assignment het slot weg?

Reply:
Het later toevoegen van een slot doet niets en zal het slot niet renderen. Indien gewenst kan dat nu al uitgebreid worden, of pas wanneer er een use case is. @kevcraey to the rescue.

Reply:
smells like yagni

Reply:
@SanderKleykens voorlopig doe ik hier niets mee, best intern met @kevcraey bespreken als je niet akkoord bent.

Reply:
Nope, was gewoon een bedenking :)